src/main/java/org/variantsync/diffdetective/variation/tree/VariationTree.java

```diff
@@ -121,6 +125,13 @@ public static VariationTree fromFile(
         return new VariationTree(tree, source);
     }
 
+    public static VariationTree fromProjection(final Projection projection, final VariationTreeSource source) {
```

Comment:
This method can be generalized to receive a `VariationNode`.

Reply:
You are right. Would it make sense though to generalize it? I am unsure what it would mean to create a projection from a variation node. That would probably be up to the variation node itself. Maybe we should leave that for a potential future refactoring? For now having a projection as argument here is more close to the paper and might serve as guidance.

Reply:
Will be refactored by @ibbem as discussed in meeting.

Reply:
I meant more of a copy, but I agree the name `fromProjection` is a special case.